docs/kb/rules/understand-severities.md

```diff
@@ -0,0 +1,20 @@
+---
+description: Understand how rule severity is determined.
+tags:
+  - Rules
+  - Semgrep Registry
+---
+
+# How does Semgrep assign severity levels to rules?
+
+## Semgrep Code
+
+Semgrep Code rules have one of three severity levels: ERROR (High), WARNING (Medium), or INFO (Low). The severity indicates how critical the issues are that a rule potentially detects.
+
+The severity of a given rule is assigned by the rule author. For custom and third-party rules, the rule author's assignment is the source of truth.
+
+As a best practice, severity for Semgrep Registry rules in the `security` category should be assigned by evaluating the combination of [likelihood](/docs/contributing/contributing-to-semgrep-rules-repository/#likelihood) and [impact](/docs/contributing/contributing-to-semgrep-rules-repository/#impact). 
```

Comment:
I feel like this instills some doubt in our severity assigning process. Why do we say "as a best practice" and "should be". Would it be accurate to just say: